packages/app/components/Queue/TA/TAQueueList.tsx

```diff
@@ -259,16 +262,46 @@ const NoQuestionsText = styled.div`
   color: #212934;
 `;
 
+function NotifReminderButton() {
+  const [isNotifOpen, setIsNotifOpen] = useState(false);
+  const NotifRemindText = styled(Button)`
+    margin-top: 16px;
+    border-radius: 6px;
+    border: 0;
+    background: #fff;
+  `;
+
+  return (
+    <>
+      <NotifRemindText onClick={(e) => setIsNotifOpen(true)}>
+        Sign Up for Notifications
+      </NotifRemindText>
+      {isNotifOpen && (
+        <NotificationSettingsModal
+          visible={isNotifOpen}
+          onClose={() => setIsNotifOpen(false)}
+        />
+      )}
+    </>
+  );
+}
+
 interface QueueProps {
   questions: Question[];
   isHelping: boolean;
   onOpenCard: (q: Question) => void;
 }
 function QueueQuestions({ questions, isHelping, onOpenCard }: QueueProps) {
+  const { phoneNotifsEnabled, desktopNotifsEnabled } = useProfile();
   return (
     <div data-cy="queueQuestions">
       {questions.length === 0 ? (
-        <NoQuestionsText>There are no questions in the queue</NoQuestionsText>
+        <>
+          <NoQuestionsText>There are no questions in the queue</NoQuestionsText>
+          {(!phoneNotifsEnabled || !desktopNotifsEnabled) && (
```

Comment:
Should we change this to check if only one of the notification types is enabled?
```suggestion
          {(!phoneNotifsEnabled && !desktopNotifsEnabled) && (
```

Don't know if everyone will choose to use both desktop and phone notifications. As long as they have one they should be good.